build.sh

```diff
@@ -20,3 +24,16 @@ if [[ $WITH_NEXT = "next" ]]; then
 else
     current
 fi
+
+# 
+# Add beacon hash to netlify.toml file
+#
+
+# Download the file
+curl -O https://static.cloudflareinsights.com/beacon.min.js
+
+# Calculate the sha256 hash of the file and convert it to base64
+hash=$(openssl dgst -sha256 -binary beacon.min.js | openssl base64)
+
+# Replace the placeholder in netlify.toml file with the calculated hash
+sed -i "s|##HASH_PLACEHOLDER##|'sha256-$hash'|g" netlify.toml
```

Comment:
This script needs to be adapted so it works with the correct hashs already inserted.